File: src/scitrader.marketdataserver/src/main/java/com/scitrader/marketdataserver/common/MarketDataServerException.java
package com.scitrader.marketdataserver.common;

public class MarketDataServerException extends RuntimeException {

        public MarketDataServerException() {
                super();
        }

        public MarketDataServerException(String message) {
                super(message);
        }

        public MarketDataServerException(String message, Exception inner){
                super(message, inner);
        }

        public MarketDataServerException(Exception inner){
                super(inner);
        }
}

File: Kotlin/hello-kotlin-world/src/main/java/com/scitrader/marketdataserver/common/MarketDataServerException.kt
package com.scitrader.marketdataserver.common

class MarketDataServerException : RuntimeException {

    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, inner: Exception) : super(message, inner) {}

    constructor(inner: Exception) : super(inner) {}
}